src/server/saved_objects/migrations/core/elastic_index.ts

```diff
@@ -52,7 +52,7 @@ export interface FullIndexInfo {
  * index mappings are somewhat what we expect.
  */
 export async function fetchInfo(callCluster: CallCluster, index: string): Promise<FullIndexInfo> {
-  const result = await callCluster('indices.get', { ignore: [404], index });
+  const result = await callCluster('indices.get', { ignore: [404], index, includeTypeName: true });
```

Comment:
I think these need to be `include_type_name` and they need to be added to the type defs... I'll do it.